### PR TITLE
Entity Creation

### PR DESCRIPTION
**Pull request by htet-oo**

### CODE REVIEW

File: src/main/java/posojt/persistence/entity/Users.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	
+	@UpdateTimestamp
+	private Date update_at;
+	
+	@OneToOne
+	@JoinColumn(name = "role_id")
+	private Roles role;
+	
+	@OneToOne(mappedBy = "user")
+	private User_profiles user_profile;
```
Comment (KhineZinThawSCM) [inline src/main/java/posojt/persistence/entity/Users.java]: update "user_profile" relation name to "userProfile"

File: src/main/java/posojt/persistence/entity/Users.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import org.hibernate.annotations.UpdateTimestamp;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "users")
+public class Users {
```
Comment (KhineZinThawSCM) [inline src/main/java/posojt/persistence/entity/Users.java]: Remove "s" in Users class

File: src/main/java/posojt/persistence/entity/User_profiles.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import org.hibernate.annotations.UpdateTimestamp;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "user_profiles")
+public class User_profiles {
```
Comment (KhineZinThawSCM) [inline src/main/java/posojt/persistence/entity/User_profiles.java]: update class name "User_profiles" to "UserProfile" , fix others

**Comment by scm-sai-zaw-myint** [comment]
ama review atine pl naw.

**Pull request merged**